pages/selfservice_mig.py

```diff
@@ -0,0 +1,14 @@
+from pages.auth0 import Auth0
+from pages.base import Base
+from pages.two_factor_authentication import TwoFactorAuthentication
+
+
+class SelfServiceMig(Base):
+    def __init__(self, selenium, url):
+        super(SelfServiceMig, self).__init__(selenium)
+        self.go_to_url(url)
+
+    def login_with_ldap(self, email, password):
```

Comment:
``login_with_ldap`` and ``__init__`` are exactly the same in both classes defined.